packages/ui/react-ui-editor/src/components/Toolbar/Toolbar.tsx

```diff
@@ -251,7 +251,7 @@ const MarkdownBlocks = () => {
   const { t } = useTranslation(translationKey);
   const value = markdownBlocks.find(({ getState }) => state && getState(state));
   return (
-    <NaturalToolbar.ToggleGroup type='single' value={value?.type}>
+    <NaturalToolbar.ToggleGroup type='multiple' value={value?.type ? [value.type] : []}>
```

Comment:
This looks redundant — `single` should be sufficient here and `type=multiple` affects how this behaves down to the HTML.

If value might be falsy, I believe you can use an empty string: `value?.type ?? ''`.

Reply:
i would have assumed single could take undefined; to me `''` IS a value.

Reply:
Since this is a controllable state prop, I think Radix does distinguish between `undefined` and `''` as signaling whether you intend to control `value` or not. Using `''` this group behaves as expected:

https://github.com/dxos/dxos/assets/855039/3e639444-8376-4d1c-a5f8-199f159d22c8

Reply:
`''` is okay as a value because it doesn’t match the id of any actual option.

Reply:
yes that's a good point re undefined and controlled values. i think `null` would be a more appropriate value though.

Reply:
an issue in your video is that the tasks button should be disabled while you're in a bullet list so the state doesn't seem to be respected (or is incorrect)?

Reply:
Yes I wasn’t quite sure what was going on there… it seems like CM is interpreting the list type as being the latest one applied, even if it might seem as if multiple are applied.